### PR TITLE
Reformats select panel

### LINKED ISSUE #39: Shrink the TokenSelector component
It has too much real estate. Shrink it down/move it/whatever

### PR DESCRIPTION
**Pull request by jdmaryville**
Reduces height between tokens, decreased width of token panel.

fixes #39

### CODE REVIEW

**Comment by thebho** [comment]
Can you provide a photo?

File: src/styles/codesplain.css [hunk 1]
```diff
@@ -4,3 +4,43 @@
   padding-top: 2px;
   padding-bottom: 2px;
 }
+
+.snippetarea {
```
Comment (Hopding) [inline src/styles/codesplain.css]: Are you using all of the CSS classes that you've defined in this file?

File: src/components/LanguageSelector.jsx [hunk 2]
```diff
@@ -19,6 +20,7 @@ const LanguageSelector = ({ languages, onChange, selected }) => {
       floatingLabelText="Language"
       onChange={onChange}
       value={selected}
+      className="styles-tokenselector"
```
Comment (dane-johnson) [inline src/components/LanguageSelector.jsx]: You should use a javascript style object here like for the others.

File: src/components/SnippetArea.jsx [hunk 3]
```diff
@@ -190,10 +184,12 @@ class SnippetArea extends React.Component {
           value={this.props.title}
           name="snippetName"
           onChange={this.props.onTitleChanged}
+          className="style-textfield"
```
Comment (dane-johnson) [inline src/components/SnippetArea.jsx]: Ditto here

File: src/components/SnippetArea.jsx [hunk 4]
```diff
@@ -190,10 +184,12 @@ class SnippetArea extends React.Component {
...
         />
         <LockButton
           onClick={this.toggleLockDialogVisibility}
           readOnly={this.props.readOnly}
+          className="style-lockbutton"
```
Comment (dane-johnson) [inline src/components/SnippetArea.jsx]: And here

File: src/components/TokenSelector.jsx [hunk 5]
```diff
@@ -19,13 +20,15 @@ class TokenSelector extends React.Component {
...
         <ListItem
+          className="style-listheight"
           key={`${tokenText}-index`}
           leftCheckbox={
             <Checkbox
               checked={tokens[token].selected}
               onCheck={
                 (ev, checked) => this.props.onChange(token, checked)
               }
+              className="styles-checkbox"
```
Comment (dane-johnson) [inline src/components/TokenSelector.jsx]: Two places in this file.

**Comment by jdmaryville** [comment]
screen shot 2017-03-03 at 10 25 51 am

File: src/components/AppBody.jsx [hunk 6]
```diff
@@ -168,14 +175,14 @@ class AppBody extends React.Component {
...
-    const infoPanelPrompt = this.state.readOnly ? 
+    const infoPanelPrompt = this.state.readOnly ?
       'Click on a line number to add an annotation or display one' :
       'Lock this snippet to add annotations';
     return (
       <div className="container-fluid">
         <div className="row">
-          <div className="col-md-3">
-            <Card>
+          <div className="col-md-2 style-selector">
```
Comment (dane-johnson) [inline src/components/AppBody.jsx]: You should make this one a JS object as well.

File: src/components/AppBody.jsx [hunk 7]
```diff
@@ -1,19 +1,26 @@
...
-
 import { Card, CardText } from 'material-ui/Card';
-
 import LanguageSelector from './LanguageSelector';
 import SnippetArea from './SnippetArea';
 import TokenSelector from './TokenSelector';
 import TokenInfoPanel from './TokenInfoPanel';
-
 import axios from 'axios'
+import '../styles/codesplain.css';
```
Comment (dane-johnson) [inline src/components/AppBody.jsx]: May not need this

File: src/components/LanguageSelector.jsx [hunk 8]
```diff
@@ -1,9 +1,15 @@
 import React, { PropTypes } from 'react';
-
+import '../styles/codesplain.css';
```
Comment (dane-johnson) [inline src/components/LanguageSelector.jsx]: Don't import this if you don't use any styles from it.

File: src/components/TokenSelector.jsx [hunk 9]
```diff
@@ -2,6 +2,7 @@ import React, { PropTypes } from 'react';
 import { List, ListItem } from 'material-ui/List';
 import Subheader from 'material-ui/Subheader';
 import Checkbox from 'material-ui/Checkbox';
+import '../styles/codesplain.css';
```
Comment (dane-johnson) [inline src/components/TokenSelector.jsx]: Ditto here

File: src/styles/codesplain.css [hunk 10]
```diff
@@ -4,3 +4,18 @@
   padding-top: 2px;
   padding-bottom: 2px;
 }
+
+.CodeMirror {
+  height: auto;
+}
+
+.style-selector {
```
Comment (dane-johnson) [inline src/styles/codesplain.css]: Make this CSS class into a JS object

**Pull request merged**